deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -376,6 +404,15 @@ Vue.component("asg-capacity-config", {
                 this.showSizeWarning = true;
             } else {
                 this.showSizeWarning = false;
+                if (maxIncrease > this.originalMaxSize  && this.originalMinSize > 0) {
+                    if (this.originalMaxSize > 100) {
+                        this.showDoubleDanger = true;
+                        this.doubleDanger = getCapacityDoubleAlertMessage(false);
```

Comment:
`getCapacityDoubleAlertMessage` in line :208 doesn't seem to have any boolean arg?

Reply:
nice catch LOL